Packs/Armis/Integrations/Armis/Armis.py

```diff
@@ -50,8 +50,6 @@ def _get_token(self, force_new: bool = False):
             response = self._http_request('POST', '/access_token/', params={'secret_key': self._secret})
             token = response.get('data', {}).get('access_token')
             expiration = response.get('data', {}).get('expiration_utc')
-            if not token:
-                raise DemistoException('Token not returned from Armis')

```

Comment:
We removed this because if the HTTP request for the token fails, an exception with the error message that returns from the server is raised and we thought the message Armis returned for the request would be the most indicative.